### DISCUSSION TITLE
using the simnibs_python in miniconda

### DISCUSSION THREAD

**Discussion by Jaycob-jh** [simnibs/simnibs#233] [6. Getting started]
When i using the command line "simnibs_python -m pip install spyder==5.3.1" in the command of macos (intel), it will show the error "zsh: command not found: simnibs_python".So, has anyone experienced the same problem before, when installing with miniconda?

**Comment by bjornsig** [comment]
Hi, we only set the simnibs python command for bash and not zsh. Can you try with bash?

**Reply by Jaycob-jh** [reply]
In fact, it still produces the same error “-bash: simnibs_python: command not found”. I'm using this command in miniconda's virtual environment simnibs_env.

**Reply by oulap** [reply]
if you have the virtual environment active the python interpreter should we configured correctly and you can just call "python" instead of "simnibs_python"
